docs/index.rst

```diff
@@ -246,6 +278,28 @@ Alternatively you can use the ``randomize`` argument to ensure a (near) unique f
 
 The  ``storage.base_url`` setting should be set to ``//s3amazonaws.com/<my-bucket-name>/`` unless you want to serve the file behind a proxy or through your Pyramid application.
 
+Usage: s3 async file storage
+----------------------
+
+.. warning::
+    S3 support requires you install the `aioboto3`_ and `aiofile`_ library separately (e.g. ``pip install aioboto3 aiofile``).
```

Comment:
Maybe we can add an extra_require rule in setup.py so that we can install it like `pip install pyramid_storage[s3async]` with the mandatory dependencies.

Reply:
Well you did actually :+1: